src/app/RegionCodeSearch.tsx

```diff
@@ -1,23 +1,25 @@
 "use client";
 
 import { FormEvent, useState } from "react";
-import { findAllRegions, Region } from "./actions";
+import { findMatchingRegions } from "./actions";
 import Link from "next/link";
 import { Button } from "@/components/ui/button";
 import { Input } from "@/components/ui/input";
 import { Label } from "@/components/ui/label";
+import { FuseResult } from "fuse.js";
+import { FlattenedRegion } from "./actions";
 
 export function RegionCodeSearch() {
   "use client";
   const [searchQuery, setSearchQuery] = useState("");
-  const [allMatchingRegions, setAllMatchingRegions] = useState<Array<Region>>(
-    [],
-  );
+  const [fuseSearchResults, setFuseSearchResults] = useState<
```

Comment:
I was wondering about this when thinking of the new names and so I'm glad you've pointed this out generally. 
Yeah, I hear you in terms of future changes to different services. 

I'll leave for example "fuseSearchMatches" for now because I think it makes it clear to us what's happening.

Reply:
> I'll leave for example "fuseSearchMatches" for now because I think it makes it clear to us what's happening.

Does that add clarity over something like `searchMatches`? Is there value, at that point in the code, to knowing that the response has been produced by something called `fuse`?

Reply:
Hmmmm, I suppose it's just a reminder? But then you'd be reminded pretty quickly if you went to actions.ts while trying to unpick how the search function worked. I will change it.